test/common/alerts_exist.go

```diff
@@ -350,7 +350,8 @@ func managedApiSpecificRules() []alertsTestRule {
 }
 
 // Common to all install types
-func commonExpectedRules() []alertsTestRule {
+func commonExpectedRules(installationName string) []alertsTestRule {
+	titledName := strings.Title(installationName)
```

Comment:
```suggestion
	upperName := strings.ToUpper(installationName)
```
Can just use initalise uppercase name here instead since there is no usage of the title name in this fucntion. But this is pretty small, so am happy for it to be addressed later